dist/tempManager.js

```diff
@@ -0,0 +1,61 @@
+class TempManager {
+    constructor() {
+        this.cityData = []
+    }
+
+    async getDataFromDB() {
+        try {
+            const cities = await $.get(`/cities`)
+            cities.forEach(c => this.cityData.unshift(c))
+            return (this.cityData)
+        } catch (err) {
+            console.log(err.message)
+        }
+    }
+
+    async getCityData(city) {
+        try {
+            const cityWeather = await $.get(`/weather/${city}`)
+            await this.cityData.unshift(cityWeather)
+        } catch (err) {
+            console.log(err.message)
+        }
+    }
+
+    async updateCity(cityName) {
+        const index = this.cityData.findIndex(c => c.name === cityName)
+        await $.ajax({
```

Comment:
try using axios in your next project. It is used a lot more than `$.ajax` and looks cleaner